spec/models/comfy/cms/page_spec.rb

```diff
@@ -251,6 +251,22 @@
     end
   end
 
+  describe '#scheduled_today scope' do
```

Comment:
Good spot. And to add more, this is a class method so it should be ".scheduled_today" as a convention.